oat-sa/extension-tao-itemqti-pci
# feat/TR-2791/math-entry-pci-support-rtl

**Related to:** [TR-2791](https://oat-sa.atlassian.net/browse/TR-2791)

**Description:**

Add support to PCI Math Entry to display on RTL mode.

**Changes:**

- Add CSS rules to support RTL mode with LTR mode compatibility.

**How to check:**
- if you using existing instance, you will need to run migration `sudo -u www-data php index.php '\oat\tao\scripts\tools\Migrations' -c migrate -v 'oat\qtiItemPci\migrations\Version202111150718531465_qtiItemPci' -e qtiItemPci`
- Create a PCI Math Entry item in RTL language. And duplicate it on LTR language.
- Create a test with both items and create TT with RTL language.
- Review test with the Guest user (LTR) and RTL TT.
- Check 4 scenarios (LTR + RTL / LTR + LTR / RTL + RTL / RTL + LTR)

**TAE Environment:**

- http://test-tr-2791.playground.kitchen.it.taocloud.org:49801/
- [credentials](http://playground.kitchen.it.taocloud.org/?p=test-tr-2791_delivery.git;a=blob;f=app/admin.txt;h=3c523e3f3f782c849e7407e8df36d6ef31a60917;hb=refs/heads/master)

**Local test:**

- https://github.com/oat-sa/kitchen-playground/blob/test/tr-2791/files/delivery/composer.json
